data/json/furniture_and_terrain/furniture-storage.json

```diff
@@ -930,5 +930,53 @@
         { "item": "splinter", "count": [ 1, 20 ] }
       ]
     }
+  },
+  {
+    "type": "furniture",
+    "id": "f_dumbbell_rack",
+    "name": "dumbbell rack",
+    "looks_like": "f_rack",
+    "description": "A metal frame designed to hold dumbbells for easy access.",
+    "symbol": "{",
+    "color": "black",
+    "move_cost_mod": -1,
+    "coverage": 10,
+    "required_str": 6,
+    "flags": [ "TRANSPARENT", "PLACE_ITEM", "MOUNTABLE", "DECONSTRUCT", "PERMEABLE", "ROUGH" ],
+    "deconstruct": { "items": [ { "item": "pipe", "count": [ 4, 6 ] } ] },
+    "max_volume": "55 L",
+    "bash": {
+      "str_min": 6,
+      "str_max": 30,
+      "sound": "metal screeching!",
+      "sound_vol": 3,
+      "sound_fail": "clang!",
+      "sound_fail_vol": 5,
+      "items": [ { "item": "scrap", "count": [ 2, 8 ] }, { "item": "steel_chunk", "count": [ 2, 4 ] }, { "item": "pipe", "count": 1 } ]
+    }
+  },
+  {
+    "type": "furniture",
+    "id": "f_plate_rack",
+    "name": "plate rack",
+    "looks_like": "f_rack",
+    "description": "A metal rack designed to hold plated shaped weights",
```

Comment:
```suggestion
    "description": "A metal rack designed to hold plated shaped weights.",
```

Reply:
10-4 buddy. I just got home and I'm on furlough all next week. I'm gonna get on this. I have a few more things to change in that as well. Thanks for pointing it out.